src/resolvers/uptimeResolvers.ts

```diff
@@ -24,7 +22,7 @@ const localizedVersionHashTitle = (name: string, language: string) => {
 };
 
 export const Query = {
-  async alerts(_: any, __: any, context: ContextWithLoaders): Promise<GQLUptimeAlert[]> {
+  async alerts(_: any, __: any, context: ContextWithLoaders): Promise<GQLUptimeAlert[] | undefined> {
```

Comment:
Kanskje sørge for at denne returnerer et tomt array istedenfor?

Reply:
Ingen mening personlig, kan godt det. Jeg bare gjorde så typene matchet det som skjer.